automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala

```diff
@@ -16,6 +16,7 @@ object LeonardoConfig extends CommonConfig {
     val pythonImageUrl: String = leonardo.getString("pythonImageUrl")
     val hailImageUrl: String = leonardo.getString("hailImageUrl")
     val gatkImageUrl: String = leonardo.getString("gatkImageUrl")
+    val aouImageUrl: String = leonardo.getString("aouImageUrl")
```

Comment:
There needs to be a corresponding entry here: https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/resources/reference.conf